packages/teams-js/src/internal/communication.ts

```diff
@@ -286,29 +299,42 @@ function updateRelationships(messageSource: Window, messageOrigin: string): void
   flushMessageQueue(Communication.childWindow);
 }
 
+const handleParentMessageLogger = communicationLogger.extend('handleParentMessage');
+
 /**@internal */
 function handleParentMessage(evt: DOMMessageEvent): void {
+  const logger = handleParentMessageLogger;
+
   if ('id' in evt.data && typeof evt.data.id === 'number') {
     // Call any associated Communication.callbacks
     const message = evt.data as MessageResponse;
     const callback = CommunicationPrivate.callbacks[message.id];
+    logger('Received a response from parent for message %i', message.id);
     if (callback) {
+      logger('Invoking the registered callback for message %i with arguments %o', message.id, message.args);
```

Comment:
Same here. For location API, e.g., the callback would consist of a user's location, correct?

Reply:
Yes, but because even if enabled it'll only log that to the browser's developer console, this should not be a concern, imo. Would you agree?

Reply:
Yes, I agree.